### PR TITLE
worked on both model and test for CowSymptom

### PR DESCRIPTION
**Pull request by NathanielWatife**
worked on both model and test for CowSymptom.

### CODE REVIEW

**Review by mamesareagan — COMMENTED** [review]
Good job👏
 
The code looks okay but you can have the choices in the choices.py and have the validations in validation.py

**Comment by NathanielWatife** [comment]
Okay,
I will do that as soon aas possible.


Thank you.

File: tests/core/tests/test_models.py [hunk 1]
```diff
@@ -28,3 +34,140 @@ def test_create_breed_with_duplicate_name(self):
...
+        assert cow_inventory.number_of_male_cows == 0
+        assert cow_inventory.number_of_female_cows == 0
+        assert cow_inventory.number_of_sold_cows == 0
+        assert cow_inventory.number_of_dead_cows == 0
+
+
+
+
+@pytest.mark.django_db
+def TestCowSymptomModel():
```
Comment (peter-evance) [inline tests/core/tests/test_models.py:139]: Use of fixtures is highly favoured, check on that please

File: core/models.py [hunk 2]
```diff
@@ -224,3 +224,61 @@ def __str__(self):
         Returns a string representation of the inseminator.
         """
         return f"{self.first_name} {self.last_name}"
+
+
+
+
+class CowSymptom(models.Model):
```
Comment (peter-evance) [inline core/models.py:231]: Validation checks should be in the respective validators.py

**Review by peter-evance — CHANGES_REQUESTED** [review]
Missing code such as the URL config, Viewset and Serializer.

**Pull request closed without merging**